### PR TITLE
update storybook fixtures

### PR DESCRIPTION
**Pull request by radium-v**
# Pull Request

## 📖 Description

Updates the Storybook fixtures to use the built-in features that Storybook provides.

### 🎫 Issues

Related to #5805. Storybook needs to have isolated fixtures for Playwright to use.

### General

- [ ] I have included a change request file using `$ yarn change`
- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://www.fast.design/docs/community/code-of-conduct/#our-standards) for this project.

## ⏭ Next Steps

#5805

### CODE REVIEW

File: packages/utilities/fast-web-utilities/src/aria.ts [hunk 1]
```diff
@@ -1,4 +1,6 @@
-export enum Orientation {
-    horizontal = "horizontal",
-    vertical = "vertical",
-}
+export const Orientation = {
+    horizontal: "horizontal",
+    vertical: "vertical",
+} as const;
+
+export type Orientation = typeof Orientation[keyof typeof Orientation];
```
Comment (scomea) [inline packages/utilities/fast-web-utilities/src/aria.ts:6]: This change feels like it should be in its own PR, or at least grouped with similar changes.  It's impact goes beyond our storybook. 
Does this risk breaking folks who have a dependency on the 'enum' type?
Wouldn't it be simpler to define the exported type like so?
`export type Orientation = "vertical" | "horizontal";`

Comment (radium-v) [reply packages/utilities/fast-web-utilities/src/aria.ts:6]: There should be no major differences between using an enum vs an object: https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums

Comment (radium-v) [reply packages/utilities/fast-web-utilities/src/aria.ts:6]: > Wouldn't it be simpler to define the exported type like so?

Explicit types can't be used as values, (ex. `Object.values(Orientation)`). It's useful when setting arg array values for Storybook controls.

**Comment by chrisdholt** [comment]
Closing as `fast-components` has been removed. We are adding stories for baseline testing and development to `fast-foundation` and we should absolutely leverage this. I have a local branch I'll push and ideally we can collaborate on that.
